art_projected_gui/src/gui/ui_core_ros.py

```diff
@@ -411,14 +417,24 @@ def object_selected(self,  id,  selected):
                 pass
 
             poly_points = []
+            square_points = []
 
             self.program_vis.set_object(obj.object_type)
             self.select_object_type(obj.object_type)
 
+            # pozicie itemov na scene (tie dva co tam su momentalne)
             for obj in self.get_scene_items_by_type(ObjectItem):
                 poly_points.append(obj.get_pos())
+                square_points.append(obj.get_pos())
+
+
+            # !vytvori polygon po dvojkliku na item! #
+            # self.add_polygon(translate("UICoreRos", "PICK POLYGON"),  poly_points,  polygon_changed=self.polygon_changed)
+
 
-            self.add_polygon(translate("UICoreRos", "PICK POLYGON"),  poly_points,  polygon_changed=self.polygon_changed)
+            # ! ja som pridal ! #
+            self.add_square(translate("UICoreRos", "PICK SQUARE GRID"),  square_points,  square_changed=self.square_changed)
```

Comment:
Ja som ho zatiaľ nahradil úplne za polygon, čiže sa mi zobrazí v ľavom rohu hneď jak sa vybere objekt, ktorý sa bude prenášať. To som sa chcel práve spýtať jak to bude prebiehať. Ako prvé sa zobrazí polygon okolo objektov, z ktorých sa bude presúvať a následne sa namiesto polygonu zobrazí ten môj grid?